### ISSUE TITLE
axe-core tries to reference node's crypto, which requires a fallback config in webpack 5

### ISSUE THREAD

**Issue by stof** [storybookjs/storybook#14538]
**Describe the bug**
When using the webpack 5 builder, the following error happens during a buildt (I haven't seen the same error when using `start-storybook` instead of `build-storybook`, which I find weird, but maybe that's why the progress stayed at 99% when using start):

```
ERR! => Failed to build the preview
ERR! Module not found: Error: Can't resolve 'crypto' in '/tmp/build_4a378f9f/node_modules/axe-core'
ERR! 
ERR! BREAKING CHANGE: webpack < 5 used to include polyfills for node.js core modules by default.
ERR! This is no longer the case. Verify if you need this module and configure a polyfill for it.
ERR! 
ERR! If you want to include a polyfill, you need to:
ERR! 	- add a fallback 'resolve.fallback: { "crypto": require.resolve("crypto-browserify") }'
ERR! 	- install 'crypto-browserify'
ERR! If you don't want to include a polyfill, you can use an empty module like this:
ERR! 	resolve.fallback: { "crypto": false }
```

**To Reproduce**

1. Add the a11y addon
1. Use the webpack5 builder
1. run `build-storybook`
1. See error

**Expected behavior**
The build works.

**System**
Storybook 6.2.7

**Additional context**
Adding `config.resolve.fallback.crypto = false` in my `webpackFinal` callback fixes the issue.

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1199622255231368/1200177927822738) by [Unito](https://www.unito.io/learn-more)

**Comment by shilman** [comment]
Do you have a repro repo you can share?

**Comment by stof** [comment]
@shilman here is the reproducer:

1. `npx sb init --type html --builder webpack5` (works also with `--type react` but it requires an extra step to add `react` and `react-dom` in the project. I haven't tried other types)
2. `npx sb add @storybook/addon-a11y`
3. edit `.storybook/main.js` to enable the a11y addon (why isn't it done by `sb add` ?)
4. run `npm run build-storybook`

**Comment by stof** [comment]
@shilman should this be closed now that https://github.com/storybookjs/storybook/pull/14592 is merged ?

**Comment by shilman** [comment]
Will close when released

**Comment by shilman** [comment]
Yo-ho-ho!! I just released https://github.com/storybookjs/storybook/releases/tag/v6.3.0-alpha.8 containing PR #14592 that references this issue. Upgrade today to the `@next` NPM tag to try it out!

```
npx sb upgrade --prerelease
```

Closing this issue. Please re-open if you think there's still more to do.

**Comment by shilman** [comment]
Ermahgerd!! I just released https://github.com/storybookjs/storybook/releases/tag/v6.3.0-alpha.26 containing PR #14914 that references this issue. Upgrade today to the `@next` NPM tag to try it out!

```
npx sb upgrade --prerelease
```
